### DISCUSSION TITLE
Thoughts on using CCXT

### DISCUSSION THREAD

**Discussion by nwillemse** [gbeced/basana#21] [Ideas]
Hi,

Have you thought about using https://github.com/ccxt/ccxt to integrate live trading with more exchanges?

Thank you

**Comment by gbeced** [comment]
Sounds like a great idea! I'll add this to my backlog and get back to you once I start making progress on this.

**Comment by gbeced** [comment]
Support for CCXT will be shipped in the next version: https://github.com/gbeced/basana/commit/2011223d5652f5c70d7ea08559deff98c39344fd
